util/settings/settings.go

```diff
@@ -133,6 +133,7 @@ type Help struct {
 type OIDCConfig struct {
 	Name                   string                 `json:"name,omitempty"`
 	Issuer                 string                 `json:"issuer,omitempty"`
+	UserInfoPath           string                 `json:"userinfopath,omitempty"`
```

Comment:
Can you write up some docs on how to use this?

Reply:
Sure, I added a section below "Requesting additional ID token claims". Is it clear what the setting does?

Reply:
lgtm!